### ISSUE TITLE
OnloadScript Myfaces 2.3.0

### ISSUE THREAD

**Issue by cocorossello** [omnifaces/omnifaces#448]
Hi,
I'm getting this error on every ajax request if I have an <o:onloadScript> defined:

```
java.lang.AbstractMethodError: org.omnifaces.component.script.OnloadScript$1.getWrapped()Ljavax/faces/context/ResponseWriter;
	at org.omnifaces.component.script.OnloadScript$1.writeText(OnloadScript.java:119)
	at org.apache.myfaces.view.facelets.compiler.LiteralTextInstruction.write(LiteralTextInstruction.java:38)
	at org.apache.myfaces.view.facelets.compiler.UIInstructions.encodeBegin(UIInstructions.java:46)
	at org.apache.myfaces.view.facelets.compiler.UILeaf.encodeAll(UILeaf.java:491)
	at javax.faces.component.UIComponentBase.encodeChildren(UIComponentBase.java:636)
	at org.omnifaces.component.script.OnloadScript.processEvent(OnloadScript.java:124)
	at javax.faces.event.SystemEvent.processListener(SystemEvent.java:49)
	at javax.faces.event.ComponentSystemEvent.processListener(ComponentSystemEvent.java:57)
	at org.apache.myfaces.application._ApplicationUtils._traverseListenerListWithCopy(_ApplicationUtils.java:168)
```

I haven't tested with latest snapshot, but I haven't seen any change so I think that the bug is still there.

I think it's a bug in myfaces since javax.faces.context.ResponseWriterWrapper class does not match the mojarra api class. I'll open an issue also there

**Comment by cocorossello** [comment]
https://issues.apache.org/jira/browse/MYFACES-4210

**Comment by BalusC** [comment]
Bug in MyFaces. Scheduled to be fixed in 2.3.1 as per https://issues.apache.org/jira/browse/MYFACES-4208

**Comment by BalusC** [comment]
https://github.com/omnifaces/omnifaces/wiki/Known-Issues-(general) has been updated.
